src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -734,13 +754,45 @@ fn run(mut args: Args) -> Result<(), anyhow::Error> {
     };
     let secrets_reader = secrets_controller.reader();
     let now = SYSTEM_TIME.clone();
-    let persist_clients = PersistClientCache::new(
-        PersistConfig::new(&mz_environmentd::BUILD_INFO, now.clone()),
-        &metrics_registry,
+
+    let persist_pubsub_addr = format!(
```

Comment:
`internal_persist_pubsub_listen_addr` will be 127.0.0.1 with the process orchestrator (everything on the same machine) and 0.0.0.0 in Kube so it's available to other pods.

The hostname is separate actually... and maybe I should rename it to `persist-pubsub-url` or `persist-pubsub-service-name` for clarity. In Kube, we'll create a Service `persist-pubsub` that internally routes to `envd`, but gives `clusterd` a stable name to talk to that's decoupled from the lifetime of an `envd` pod, or even having to know that the server is hosted by `envd`

Reply:
so in Kube, we'd expect a Service `persist-pubsub` that routes to `envd` and then to set:

`internal_persist_pubsub_listen_addr: 0.0.0.0:6879` - so that `envd` listens to connections on 6879
`persist-pubsub-hostname: persist-pubsub` - so that `clusterd` knows to connect to `persist-pubsub:6879` (which happens to be served by `envd`) as the server